src/test/java/com/google/api/codegen/testdata/ruby_doc_main_library.baseline

```diff
@@ -810,6 +810,7 @@ module Library
       # @example
       #   require "library/v1/library_service_api"
       #
+      #   Alignment = Google::Example::Library::V1::SomeMessage2::SomeMessage3::Alignment
```

Comment:
Yep. Some form of disambiguation will probably be needed. I don't think this is breaking though. The RubyTypeTable in MVVM accounts for scenarios where [shortnames](https://github.com/googleapis/toolkit/blob/master/src/main/java/com/google/api/codegen/util/ruby/RubyTypeTable.java#L87) are already used.